.github/workflows/ghcr.yml

```diff
@@ -98,31 +192,43 @@ jobs:
       - name: Install Python dependencies using Poetry
         run: make install-python-dependencies
 
-      - name: Download sandbox Docker image
+      - name: Download Runtime Docker image
+        if: matrix.runtime_type == 'eventstream'
+        uses: actions/download-artifact@v4
+        with:
+          name: od_runtime-docker-image-amd64
```

Comment:
`amd64` should be the matrix value (also in below step)?

Reply:
oh, `test_runtime` is amd64 only?

Reply:
yeah, we are running tests for amd64 for now. But indeed we can test on arm64 as well.